### PR TITLE
Update 0.3.2

### PR DESCRIPTION
**Pull request by Shawdooow**

### CODE REVIEW

File: osu.Game.Rulesets.Vitaru/Objects/Drawables/DrawablePatterns.cs [hunk 1]
```diff
@@ -18,7 +18,7 @@ public class DrawablePattern : DrawableVitaruHitObject
         private bool patternCreated = false;
 
         public int BulletCount = 0;
-        public int PatternCount = 0;
+        public static int PatternCount = 0;
```
Comment (Jorolf) [inline osu.Game.Rulesets.Vitaru/Objects/Drawables/DrawablePatterns.cs]: why?

Comment (Arrcival) [inline osu.Game.Rulesets.Vitaru/Objects/Drawables/DrawablePatterns.cs]: cause if not it's always 0 ?

Comment (Jorolf) [inline osu.Game.Rulesets.Vitaru/Objects/Drawables/DrawablePatterns.cs]: I meant why he made it static

Comment (Shawdooow) [inline osu.Game.Rulesets.Vitaru/Objects/Drawables/DrawablePatterns.cs]: so its not always 0?

Comment (ColdVolcano) [inline osu.Game.Rulesets.Vitaru/Objects/Drawables/DrawablePatterns.cs]: I don't even know if we want to have this here or in the playfield

Comment (Arrcival) [inline osu.Game.Rulesets.Vitaru/Objects/Drawables/DrawablePatterns.cs]: It's no more useful since score result will always be 300/Miss

Comment (ColdVolcano) [inline osu.Game.Rulesets.Vitaru/Objects/Drawables/DrawablePatterns.cs]: Why do we assign only 300/miss, I'm pretty sure next update will be the score update

**Pull request merged**